docker/templates/Dockerfile.j2

```diff
@@ -45,13 +53,18 @@ ENV PATH=/usr/share/logstash/bin:$PATH
 # Provide a minimal configuration, so that simple invocations will provide
 # a good experience.
 ADD config/pipelines.yml config/pipelines.yml
-ADD config/logstash-{{ image_flavor }}.yml config/logstash.yml
+{% if image_flavor == 'oss' -%}
+ADD config/logstash-oss.yml config/logstash.yml
+{% else -%}
+ADD config/logstash-full.yml config/logstash.yml
+{% endif -%}
 ADD config/log4j2.properties config/
 ADD pipeline/default.conf pipeline/logstash.conf
 RUN chown --recursive logstash:root config/ pipeline/
 
 # Ensure Logstash gets a UTF-8 locale by default.
-ENV LANG='en_US.UTF-8' LC_ALL='en_US.UTF-8'
+# Minimal distributions do not ship with en language packs.
+ENV LANG='C.UTF-8' LC_ALL='C.UTF-8'
```

Comment:
I wonder if it makes sense that we only to set 'C.UTF-8' for ubi and use 'en_US.UTF-8' for the full one?

Reply:
Also, CI is failing.